packages/kit/src/core/sync/write_non_ambient.js

```diff
@@ -99,7 +101,7 @@ function generate_app_types(manifest_data) {
 		`\t\tLayoutParams(): {\n\t\t\t${layouts.join(';\n\t\t\t')}\n\t\t};`,
 		`\t\tPathname(): ${Array.from(pathnames).join(' | ')};`,
 		'\t\tResolvedPathname(): `${"" | `/${string}`}${ReturnType<AppTypes[\'Pathname\']>}`;',
-		`\t\tAsset(): ${manifest_data.assets.map((asset) => s('/' + asset.file)).join(' | ') || 'never'};`,
+		`\t\tAsset(): ${assets.concat('string & {}').join(' | ')};`,
```

Comment:
"plus a `/${string}` wildcard for asset paths generated from `import` declarations" -> this isn't the case here

```suggestion
		`\t\tAsset(): ${assets.concat('`/${string}` & {}').join(' | ')};`,
```

Reply:
it was originally this, but it doesn't work because the type of a generated asset string is just `string`, and you can't assign a `string` to `/${string}`. i'll fix the docs instead